pkg/daemon/controller/controller.go

```diff
@@ -419,15 +419,17 @@ func (sc *ServiceController) resolveNodeVariables(svc servicescm.Service) (map[s
 }
 
 // resolvePowershellVariables returns a map, with the keys being each variable, and the value being the string to
-// replace the variable with
+// replace the variable with. Variables with blank names will not result in a map entry, but their script will be run.
 func (sc *ServiceController) resolvePowershellVariables(svc servicescm.Service) (map[string]string, error) {
 	vars := make(map[string]string)
 	for _, psVar := range svc.PowershellVariablesInCommand {
 		out, err := sc.psCmdRunner.Run(psVar.Path)
 		if err != nil {
 			return nil, errors.Wrapf(err, "could not resolve PowerShell variable %s", psVar.Name)
 		}
-		vars[psVar.Name] = strings.TrimSpace(out)
+		if psVar.Name != "" {
```

Comment:
An empty ` psVar.Name` will also ignore the output of the executed command/script. Do you anticipate any side effects?

Reply:
As of now I don't see any side effects. The assumption is if we supply an empty var name, we don't care for the output, just that the script executes successfully (i.e. purely pre-config for a service to start). Errors running the PS command/script will still be caught and returned.